src/applications/caregivers/tests/unit/components/ApplicationDownloadLink.unit.spec.js

```diff
@@ -75,119 +75,104 @@ describe('CG <ApplicationDownloadLink>', () => {
       recordEventStub.restore();
     });
 
-    it('should record success event when button is clicked', async () => {
-      const { container } = subject();
-      const loadingIndicator = container.querySelector('va-loading-indicator');
-
-      const link = $('va-link', container);
```

Comment:
The default test checks that the appropriate elements exists. No need to do it in all the subsequent tests.